server/routes/userInfo.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../utils/dbConfig");
+
+router.get("/:id", (req, res) => {
+    knex("users")
+      .select("users.id","users.username","users.email","users.first_name","users.last_name",
```

Comment:
when you are connecting only one table you don't have to use the prefix "users.id" or "users.username", you can simply use them as "id" and "username" because it is understood it is from that same table. You need to use the prefix when you are joining multiple tables.